daemon/commands.c

```diff
@@ -381,9 +429,12 @@ static void parse_commands(struct command_context *cmd_ctx)
     for (pos = cmd_ctx->command_string ; isspace(*pos) && ('\0' != *pos) ; ++pos) {;}
     for (i = 0 ; i < CMD_TOTAL_COMMANDS ; ++i) {
         if (!strncmp(pos, command_info_array[i].cmd_str, strlen(command_info_array[i].cmd_str))) {
+            for (lstrip=pos + strlen(command_info_array[i].cmd_str); isspace(*lstrip) && ('\0' != *lstrip); ++lstrip) {;}
+            for (rstrip=lstrip+strlen(lstrip)-1; isspace(*rstrip) && rstrip>lstrip; *(rstrip--) = 0 );
```

Comment:
if `lstrip` is empty then `strlen(lstrip)` is `0` and this may prove problematic I think.

Reply:
On the other hand, even if `rstrip` is `lstrip - 1` `*rstrip` will still land on valid memory because there exists a command in line 431, so I guess it's ok after all.

Reply:
Unresolving while I take a look, it's a minimal change and it is more important that it is correct than it doesn't do any damage.

Reply:
It would not have broken but it was a little messy so I fixed it anyway.